packages/babel-plugin-strip-runtime/src/__tests__/strip-runtime-source-code.test.ts

```diff
@@ -247,6 +247,293 @@ describe('babel-plugin-strip-runtime using source code', () => {
         );
```

Comment:
Can we add a test where both `@emotion/react` and `@compiled/react` co-exist in the same file?

Reply:
Hmm explicitly importing `@emotion/react` and `@compiled/react` in the same file isn't something we support 😅
especially given that which library should parse `css={{ color: 'blue' }}` is currently a bit ambiguous (also see [eslint rule](https://github.com/atlassian-labs/compiled/tree/master/packages/eslint-plugin/src/rules/jsx-pragma))

If what you mean is Compiled `xcss` and Emotion `css` being used in the same file, happy to look into adding a test for that

Reply:
> Hmm explicitly importing @emotion/react and @compiled/react in the same file isn't something we support 😅
especially given that which library should parse css={{ color: 'blue' }} is currently a bit ambiguous (also see [eslint rule](https://github.com/atlassian-labs/compiled/tree/master/packages/eslint-plugin/src/rules/jsx-pragma))

Got it. I forgot about it

Reply:
Can we throw instead of relying on implicit behaviour?

Reply:
> Unless the behaviour is remove jsx pragma when using jsx pragma from compiled then all good.

yup thats correct

Reply:
> If what you mean is Compiled xcss and Emotion css being used in the same file, happy to look into adding a test for that

ah just realised such a test exists already haha

https://github.com/atlassian-labs/compiled/blob/master/packages/babel-plugin/src/xcss-prop/__tests__/transformation.test.ts#L340